### DISCUSSION TITLE
SB 5.1 OCI Postgres provisioning jobs query

### DISCUSSION THREAD

**Discussion by subhashchandrab** [Sunbird-Ed/Community#413] [Installation] [answered]
Following the documentation https://ed.sunbird.org/use/installing-ed-platform/release-5.1.0/build
We noticed that the documentation contains the following jobs for Postgres provisioning 
Provision/Core/Postgres
Provision/Core/PostgresDbUpdate
However, As per the Jenkins setup and the code base https://github.com/project-sunbird/sunbird-devops/tree/release-5.1.0 we also noticed the following job which is not available in the documentation https://ed.sunbird.org/use/installing-ed-platform/release-5.1.0/build
 Provision/Core/PostgresDbUpdateManaged

Can anyone please clarify whether both the jobs are meant for different scenarios based on how the postgres is setup(like VM based vs managed service) or PostgresDbUpdateManaged is really being used in the overall setup process or now

**Comment by saiakhil46** [comment, marked as answer]
@subhashchandrab PostgresDbUpdate is for postgres installed on VM and PostgresDbUpdateManaged is for managed service in respective could service provider, please use the job based on how postgres is provisioned in your environment.
